e2e_samples/dataset_versioning/insert_sql/process.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+from datetime import datetime
+
+
+class Process:
+    """Process object to filter df with specified version
+    """
+    def __init__(self, df: pd.DataFrame, water_mark='issue_id'):
```

Comment:
Since we aren't configuring the watermark at this time (or the dataset for that matter). It may be misleading to allow `water_mark` to be configurable.

Reply:
I agree, naming is a bit confusing.

Reply:
Well I mean why even have the parameter (`water_mark`)? Is it just for testing?

Reply:
umm.. I try to make is so that only ```read_csv()```  is highly coupled with the load data. 
parameterzing water_mark (changed to version date) is required because testing sample does not have (don't want to have) issue_id

Reply:
User of this script using different sample code probably need to re-write read_csv but only read_csv.